org-super-agenda.el

```diff
@@ -505,14 +505,18 @@ COMPARISON should be a symbol, one of: `past' or `before',
 ;;;;; Effort
 
 (cl-defmacro org-super-agenda--defeffort-group (name docstring &key comparator)
+  "Define an effort-grouping function.
+NAME is a symbol appended to `effort' to create the group name.
+DOCSTRING is a string used for the function's docstring.
+COMPARATOR is a grouping comparison operator."
   (declare (indent defun))
   `(org-super-agenda--defgroup ,(intern (concat "effort" (symbol-name name)))
      ,(concat docstring "\nArgument is a time-duration string, like \"5\" or \"0:05\" for 5 minutes.")
      :section-name (concat "Effort " ,(symbol-name name) " "
                            (s-join " or " args) " items")
-     :let* ((effort-minutes (org-duration-string-to-minutes (car args))))
+     :let* ((effort-minutes (org-duration-to-minutes (car args))))
      :test (when-let ((item-effort (org-find-text-property-in-string 'effort item)))
-             (,comparator (org-duration-string-to-minutes item-effort) effort-minutes))))
+             (,comparator (org-duration-to-minutes item-effort) effort-minutes))))
```

Comment:
Please remove these function name change hunks for now.  They may break compatibility with older versions of Org, which I'm not ready to deal with.